pkg/cmd/dev/test.go

```diff
@@ -83,21 +77,16 @@ pkg/kv/kvserver:kvserver_test) instead.`,
 	// or only on failures. `--show-logs` by contrast is a flag for the process
 	// under test, controlling whether the process-internal logs are made
 	// visible.
-	testCmd.Flags().BoolP(vFlag, "v", false, "show testing process output ("+
-		"equivalent to passing '--testargs=test.v -- --test_output all')")
+	testCmd.Flags().BoolP(vFlag, "v", false, "show testing process output")
 	testCmd.Flags().Int(countFlag, 1, "run test the given number of times")
 	testCmd.Flags().BoolP(showLogsFlag, "", false, "show crdb logs in-line")
 	testCmd.Flags().Bool(stressFlag, false, "run tests under stress")
-	testCmd.Flags().String(stressArgsFlag, "", "additional arguments to pass to stress.\n"+
-		"Prefix each arg with '-'; e.g. '--stress-args='-maxtime=30s -maxruns=100'.\n"+
```

Comment:
@irfansharif your commit reverts the additional language I added to each flag description. Was this intentional?

Reply:
Yup, this was moved to the top level help text with examples. Before:

```
      --stress-args string   additional arguments to pass to stress.
                             Prefix each arg with '-'; e.g. '--stress-args='-maxtime=30s -maxruns=100'.
                             See all stress args at https://github.com/cockroachdb/stress/blob/master/main.go
      --test-args string     additional arguments to pass to the go test binary.
                             Pass each arg with '-test.{arg}'; for example: --test-args='-test.timeout 1s'.
                             Learn about go test args by running 'go help test'.
```

Now:

```
Examples:

        dev test
        dev test pkg/kv/kvserver --filter=TestReplicaGC* -v --timeout=1m
        dev test pkg/... -v
        dev test pkg/spanconfig/... pkg/ccl/spanconfigccl/...                # test multiple packages recursively
        dev test --race --stress ...                                         # run a test under race and stress
        dev test pkg/spanconfig/... --test-args '-test.trace=trace.out'      # pass arguments to the go test binary (see 'go help testflag')
        dev test pkg/spanconfig --stress --stress-args '-maxruns 1000 -p 4'  # pass arguments to github.com/cockroachdb/stress
        dev test --stress --timeout=1m --test-args='-test.timeout 5s'        # stress for 1m until a test runs longer than 5s
        dev test pkg/server -f=TestSpanStatsResponse -v --count=5 --vmodule='raft=1'
```

Reply:
works for me! Is it cool if i move the description above each example, as I had originally? I think its best to keep each line under 70 chars. The help msg currently looks like this in a small terminal window:
<img width="840" alt="image" src="https://user-images.githubusercontent.com/35438895/172501382-a2d54bdf-0661-48f7-970b-703f5c45ce2c.png">

Reply:
Works for me. An alternative is perhaps making this look like like `dev -h`, but I'm getting too bikesheddy.

Reply:
Good idea. Merged your pr. Let me know if it looks good!